src/content/docs/infrastructure/amazon-integrations/get-started/introduction-aws-integrations.mdx

```diff
@@ -121,14 +51,10 @@ This section directs you to relevant AWS docs that explain how our integrations
 * When you enable AWS Config to enrich CloudWatch metrics with resource metadata, you're charged based on the number of configuration items recorded. See the [AWS Config pricing page](https://aws.amazon.com/config/pricing/) for details.
 * [AWS Resource Groups Tagging API](https://docs.aws.amazon.com/resourcegroupstagging/latest/APIReference/overview.html) enriches metrics with custom tags defined in the associated AWS resources. This affects both polling and metric streams.
 
-New Relic uses the Amazon CloudWatch API to obtain metrics from your AWS services when you use the API Polling integration instead of CloudWatch Metric Streams. Some situations may increase the number of calls to the CloudWatch API to exceed the <DNT>**1 million free**</DNT> limits granted by AWS and [increase your CloudWatch bill](/docs/infrastructure/amazon-integrations/troubleshooting/cloudwatch-billing-increase). This may happen when you: 
-
-* Enable additional integrations through API Polling
-* Add AWS resources to those integrations
-* Scale those integrations across more regions.
+New Relic uses the Amazon CloudWatch API to capture data about AWS services when you use the API Polling integration instead of CloudWatch Metric Streams. Some situations may increase the number of calls to the CloudWatch API to exceed the <DNT>**1 million free**</DNT> limits granted by AWS and [increase your CloudWatch bill](/docs/infrastructure/amazon-integrations/troubleshooting/cloudwatch-billing-increase). This may happen when you enable additional integrations through API Polling, add AWS resources to those integrations, or scale those integrations across more regions.
```

Comment:
Maybe a link to the API Polling integration here?

Reply:
I don't think API Polling should be capitalized in the last sentence (the Polling part)